src/SigSpec.CodeGeneration.TypeScript/Templates/Hub.liquid

```diff
@@ -1,4 +1,8 @@
 ﻿export class {{ Name }}Hub {
+{% for operation in Callbacks -%}
+    callbackFor{{ operation.Name }}: ({% for parameter in operation.Parameters %}{{ parameter.Name }}: {{ parameter.Type }}{% if forloop.last == false %}, {% endif %}{% endfor %}) => void;
```

Comment:
Add private?

Reply:
To be safe this should be an array of registrations so if multiple impls are registered all callbacks are unregistered, not only the last one. But we can also leave it as is as this is an edge case.

Reply:
Modifications were made following your comments. Please check.